src/components/academies/RedirectToAcademy.jsx

```diff
@@ -0,0 +1,36 @@
+import { Navigate } from 'react-router-dom';
+import { Alert, Container } from '@openedx/paragon';
+import { Warning } from '@openedx/paragon/icons';
+import React from 'react';
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+import useEnterpriseCustomer from '../app/data/hooks/useEnterpriseCustomer';
+import { useAcademies } from '../app/data';
+
+const RedirectToAcademy = () => {
+  const { data: academies, isError: fetchError } = useAcademies();
+  const { data: enterpriseCustomer } = useEnterpriseCustomer();
+
+  if (fetchError || academies.length !== 1) {
+    return (
+      <Container size="lg" className="search-results">
+        <br />
+        <Alert
+          variant="danger"
+          dismissible={false}
+          icon={Warning}
+          open
+        >
+          <FormattedMessage
+            id="enterprise.dashboard.tab.courses.error.message"
+            defaultMessage="An error occurred while trying to load your Academy."
+            description="Error message shown to a learner on enterprise dashboard."
+          />
+        </Alert>
+      </Container>
+    );
+  }
+
+  return <Navigate to={`/${enterpriseCustomer.slug}/academies/${academies[0].uuid}`} replace />;
```

Comment:
[inform] To make it easier to access the relevant enterprise customer's metadata within the route loader, we just [merged](https://github.com/openedx/frontend-app-learner-portal-enterprise/pull/1056) a slight refactor to change `extractEnterpriseId` to `extractEnterpriseCustomer` so it returns all customer metadata (including `.enableOneAcademy`) instead of just the UUID.